### PR TITLE
Subpop pref

### PR DESCRIPTION
**Pull request by JamesKunstle**
Update to #230 ahead of base notebook revision.

Creates a rough formulation that measures the interest our sub-population of interest has in repos w.r.t the interest of the general population in the same repos.

Attempting to create a metric that is higher when our sub-population is disproportionately interested in a repo, and the interest is strong (many events) and lower both when the sub-population's interest is reasonable compared to the general popularity of the repo, and when the interest is relatively weak (few events.)

### CODE REVIEW

**Comment by JamesKunstle** [comment]
Is identical to #230 until the final markdown cell. 

Since #230 hasn't been reviewed/merged, new branch includes more changes to notebook that can be reviewed independently from analysis done in #230.

File: notebooks/project_discovery/repo_discovery.ipynb [hunk 1]
```diff
@@ -0,0 +1,10052 @@
+{
```
Comment (cdolfi) [inline notebooks/project_discovery/repo_discovery.ipynb:1]: <p>For the definitions in 1: </p><p>Γ is this the population of events for the projects in the set? </p><p> </p><p>Overall really like this. Makes my theory math brain v happy lol </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (JamesKunstle) [reply notebooks/project_discovery/repo_discovery.ipynb:1]: Made a note using Mu as the real population, and Gamma as the recorded population which is hopefully a strong approximation of Mu

**Review by cdolfi — APPROVED** [review]
lgtm! will wait for @hemajv and @oindrillac for reviews before merging

File: notebooks/project_discovery/project_discovery.ipynb [hunk 2]
```diff
@@ -0,0 +1,2122 @@
+{
```
Comment (oindrillac) [inline notebooks/project_discovery/project_discovery.ipynb:1]: <p>We do not need this file as repo_discovery is doing the same things described here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: notebooks/project_discovery/repo_discovery.ipynb [hunk 3]
```diff
@@ -0,0 +1,10542 @@
...
```
Comment (oindrillac) [inline notebooks/project_discovery/repo_discovery.ipynb:1]: <p>@JamesKunstle really like the description here. Can you also highlight some of the assumptions/limitations we discussed in call here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

File: notebooks/project_discovery/project_discovery.ipynb (hunk 2 above)
Comment (JamesKunstle) [reply notebooks/project_discovery/project_discovery.ipynb:1]: Yeah I'm not going to include this file in the final merge, it's just there temporarily.

File: notebooks/project_discovery/repo_discovery.ipynb (hunk 3 above)
Comment (JamesKunstle) [reply notebooks/project_discovery/repo_discovery.ipynb:1]: Yes certainly, will update. Thank you for the reminder.

**Comment by JamesKunstle** [comment]
I'm closing this PR because its contents are in an overleaf document and the notebook has diverged in PR #230 .

**Pull request closed without merging**